.uberenv_config.json

```diff
@@ -8,7 +8,7 @@
 "spack_configs_path": "scripts/spack/configs",
 "spack_packages_path": "scripts/spack/packages",
 "vcpkg_url": "https://github.com/microsoft/vcpkg",
-"vcpkg_commit": "b79f7675aaa82eb6c5a96ae764fb1ce379a9d5d6",
+"vcpkg_commit": "5568f110b509a9fd90711978a7cb76bae75bb092",
```

Comment:
Points to this release of `vcpkg`: https://github.com/microsoft/vcpkg/releases/tag/2021.05.12